app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt

```diff
@@ -1,67 +0,0 @@
-package com.aptoide.android.aptoidegames.drawables.icons
-
-import androidx.compose.foundation.Image
-import androidx.compose.foundation.layout.size
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.SolidColor
-import androidx.compose.ui.graphics.vector.ImageVector
-import androidx.compose.ui.graphics.vector.path
-import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.unit.dp
-import com.aptoide.android.aptoidegames.theme.primary
-
-@Preview
-@Composable
-fun TestWifi() {
-  Image(
-    imageVector = getWifi(),
-    contentDescription = null,
-    modifier = Modifier.size(240.dp)
-  )
-}
-
-fun getWifi(): ImageVector = ImageVector.Builder(
-  name = "Wifi",
-  defaultWidth = 88.dp,
-  defaultHeight = 88.dp,
-  viewportWidth = 88f,
-  viewportHeight = 88f,
-).apply {
-  path(
-    fill = SolidColor(primary),
-  ) {
-    moveTo(23.2833f, 56.2834f)
-    lineTo(15.5833f, 48.4f)
-    curveTo(19.3722f, 44.6111f, 23.7111f, 41.7084f, 28.6f, 39.6917f)
-    curveTo(33.4889f, 37.675f, 38.6222f, 36.6667f, 44f, 36.6667f)
-    curveTo(49.3778f, 36.6667f, 54.5264f, 37.6903f, 59.4458f, 39.7375f)
-    curveTo(64.3653f, 41.7848f, 68.6889f, 44.7334f, 72.4167f, 48.5834f)
-    lineTo(64.7167f, 56.2834f)
```

Comment:
Isn't this one necessary ?

Reply:
This one is not used and will not be used